homework10/kittyBall/kittyDot/ViewController.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ViewController.swift
+//  kittyDot
+//
+//  Created by Ernest Mihasenko on 1.04.22.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    @IBOutlet weak var ball: UIButton!
+  
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        ball.layer.cornerRadius = 55
+        ball.layer.borderWidth = 15
+        // Do any additional setup after loading the view.
+    }
+
+    @IBAction func Balltap(_ sender: Any) {
+        
+        ball.layer.position = CGPoint(x: Int.random(in: 45..<Int(view.frame.maxX)), y: Int.random(in: 45..<Int(view.frame.maxY)))
```

Comment:
You shoudn't use `layer.position`, you should use `frame.origin`. You shouldn't use const values here as well. You can implement it the same way I've written above